enterprise_access/apps/api/v1/views.py

```diff
@@ -1241,7 +1241,9 @@ def can_redeem(self, request, enterprise_customer_uuid=None):
             can_redeem_for_content_response = {
                 "content_key": content_key,
                 "redemptions": redemptions,
-                "subsidy_access_policy": serialized_policy,
+                "has_successful_redemption": any(redemption['state'] == 'committed' for redemption in redemptions),
```

Comment:
Hm, "committed" is a magic string, but enterprise-access doesn't install openedx-ledger (which gives us openedx_ledger.models.TransactionStateChoices) so maybe just set this at the top of the file?

```python
TRANSACTION_STATE_COMMITTED = "committed"
```